scripts/download-vscode.js

```diff
@@ -50,8 +55,15 @@ downloadAndUnzipVSCode(vscodeVersion)
     // Example:
     // ~/salesforcedx-vscode/packages/salesforcedx-vscode-lwc> npm run test:vscode-integration
     if (vscodeBasePath.indexOf(path.sep + 'packages' + path.sep) !== -1) {
+      console.log('### vscode package install found');
       // Do nothing, vscode is already downloaded and extracted in this package
+
+      const userData = path.join(vscodeBasePath, VSCODE_TEST, USER_DATA);
+      console.log('### userdata: ' + userData);
+      // Remove the previously generated user data to ensuer we can run the int tests
+      fs.rmSync(userData, { recursive: true, force: true });
```

Comment:
This is probably fallout from the upgrade.  basically there a user-data directory that gets created in the .vscode-test dir. It points a directory that causes issues on re-runs.  This fixes that problem.  I'll fix the ensuer typo in the comment above.